mysql/tests/test_statements.py

```diff
@@ -30,15 +30,23 @@
 # depend on a default schema being set on the connection
 DEFAULT_FQ_SUCCESS_QUERY = "SELECT * FROM information_schema.TABLES"
 
+CLOSE_TO_ZERO_INTERVAL = 0.0000001
```

Comment:
Is there a difference between `0.0000001` and `0`? Why not just zero?

Reply:
0 doesn't work cause we divide by 0 when we pass this argument to DBMAsync. 
So, in code there is a check that will revert 0 to default_collection_interval.
like [https://github.com/DataDog/integrations-core/blob/13b7646c076108d76194c67e80c57515564fd489/sqlserver/datadog_checks/sqlserver/statements.py#L212](url) .

Reply:
Got it, thanks!